features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -2566,6 +2576,130 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                     </>
                 ) }
 
+            { /* Legacy Application Tokens */ }
+            {
+                (!omitUsernameInIntrospectionRespForAppTokens
+                || !useClientIdAsSubClaimForAppTokens)
+                && (
+                    <Grid.Row columns={ 2 }>
+                        <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                            <Divider />
+                            <Divider hidden />
+                        </Grid.Column>
+                        <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                            <Heading as="h4">
+                                { t("applications:forms.inboundOIDC.sections" +
+                                    ".legacyApplicationTokens.heading") }
+                            </Heading>
+                            <Divider hidden />
+                            {
+                                (
+                                    <Alert severity="warning">
+                                        <AlertTitle
+                                            className="alert-title">
+                                            <Trans components={ { strong: <strong/> } } >Note:</Trans>
+                                        </AlertTitle>
+                                        <Trans
+                                            i18nKey={ t("actions:fields.authentication.info.message") }
+                                        >
+                                            You currently using an outdated behavior for application tokens.
```

Comment:
Using `you` in UI instructions is generally not recommended. Hope we can get the text reviewed with the doc team.

cc: @himeshsiriwardana